source/api/annex/notes/editors.md

```diff
@@ -0,0 +1,141 @@
+
+__N.B.:__ The current editors will help new editors with this process as necessary. Prior experience with Git or Github _MUST NOT_ be considered prerequisite knowledge when considering new editors.
+
+### 3.3 Acceptance Criteria for Merging Changes
+
+Final voting _MUST_ take place on a pull request. Non-normative editorial discussion, e.g. to point out changes in language and typos, may also transpire on the pull request, however, any discussion of substantive changes related to the issue _MUST_ continue on the issue.
+
+In addition to adhering to the guidelines above, there _MUST_ be agreement about the change among the editors. _Agreement_ is defined as follows:
+
+ * There _MUST NOT_ be any "-1"s (or similar Emoji) in the latest comment from any editor before a merge.
+ * __A majority__ of the editors, including the issuer of the pull request, _MUST_ actively agree ("+1") on any __non-normative__ change to a specification.
+ * __All but one__ of the editors _MUST_ actively agree on any __normative__ change to a specification.
+ * Even for non-specification content, __at least one__ editor, in addition to the issuer of the pull request, if applicable, must actively agree to any change on the [iiif.io][iiif-io] website.
+
+The editor that submitted the pull request _MUST NOT_ be the editor that merges it.
+
+In the event that an editor disagrees with a merge that has already happened, they should create a new issue that references the change in question, and explains the objection. This issue then serves as the discussion point for the objection which is subject to this same process. Reverts _SHOULD_ stay in the repository history, i.e., the process of reverting involve updating the HEAD version to reflect older content, and not erasing or re-writing history.
+
+### 3.4 Face to Face Meetings
+
+Editors will meet at least four times per year, and all editors _MUST_ attend at least three of those meetings per calendar year. Every effort will be made to co-locate at least one of the meetings with another meeting at which most of editors are already likely to be present.
+
+The editors _MUST_ provide a summary of all meetings, and _SHOULD_ provide a summary of other interactions (e.g. closed conference calls leading up to a specification review period) at which changes and new specifications are discussed.
+
+## 4. Editorial Team Membership and Selection
+
+Per the IIIF Memorandum of Understanding covering the establishment of an IIIF Consortium, the editors may nominate additional editors at any time. All current editors _MUST_ agree before an invitation is issued. Editors will chose from participants within the community and/or experts willing to lend their knowledge and experience to new specifications. Opportunities to better the gender, race, and age balance of the existing editorial team will also be a consideration. New editors _MUST_ be approved by the IIIF Executive Group.
```

Comment:
I think we also agreed that candidate editors __SHOULD__ have made successful changes via github, following the protocol defined.